metamorph/metamorph_plugin.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import logging.config
+import json
+import time
+import os
+
+import requests
+
+
+class MetamorphPlugin(object):
+
+    MINUTE = 60
```

Comment:
I agree with your complaint that `self.MINUTE + self.MINUTE` it is not good behavior but this is not the case right now. My thought is to avoid 'random' numbers in code and replace it with constants.